main.py

```diff
@@ -1,8 +1,39 @@
 from fastapi import FastAPI
+import requests
+import random
 
 app = FastAPI()
 
 
+# Homepage. Directs you to '/facts' page or '/breeds' page, where either you get a cat fact pulled from the
+# catfact.ninja 'facts' get request, or a breed from the 'breeds' get request"
+# https://catfact.ninja/#/
 @app.get("/")
 async def root():
-    return {"message": "Hello World"}
+    return "Random fact (from a list) generated at '/facts'. Random breed generated at '/breeds'."
```

Comment:
You can remove this root endpoint, you don't need it. Unless you wanna give it a purpose to exist.

Reply:
@rafaelbrant My thinking is this is the (route? address?) the container opens up to, so I'd rather have the above message than `{"detail":"Not Found"}`, which is what's shown when you remove the function